### PR TITLE
Add some hints on how to introspect and debug KubeVirt

### PR DESCRIPTION
**Pull request by rmohr**

### CODE REVIEW

File: docs/debugging.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+to talk to the cluster through that proxy. So if
+
+```bash
+cluster/kubectl.sh version
+```
+
+is run, `cluster/kubectl.sh` connects to the proxy. However when something is
+not right with the proxy, or KubeVirt is not even yet deployed, it is possible
+to connect to the apiserver directly by adding `--core` as the first argument.
+So
```
Comment (stu-gott) [inline docs/debugging.md]: trailing whitespace

File: docs/debugging.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+This way it is pretty easy to detect if a Pod or a VM got started.
+
+## Entering Containers
+
+It can be very valuable to enter a container and do some investigations there,
+to see what is going wrong. In this case the kubectl `exec` command can be
+used. To enter `virt-api` with an interactive shell, type
+
+```bash
+cluster/kubectl.sh exec virt-api -c virt-api -i -t -- sh
```
Comment (stu-gott) [inline docs/debugging.md]: here too

File: docs/debugging.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+the `--previous` flag can be used. To view the logs of `virt-api` from the
+previous run, type
+
+```bash
+cluster/kubectl.sh logs virt-api -f -p
+```
+
+## Watching Events
+
+Both, kubernetes and KubeVirt are creating events, which can be viewed via
```
Comment (stu-gott) [inline docs/debugging.md]: I think Kubernetes is capitalized.

**Comment by rmohr** [comment]
@stu-gott thx for reviewing. Typos fixed.

**Pull request merged**